### DISCUSSION TITLE
Combine Tooltip with the Dialog

### DISCUSSION THREAD

**Discussion by Marsony** [floating-ui/floating-ui#2248] [Q&A]
Hello everyone! And again ty very much for this awesome library!

So, is it possible to combine few features?

I got following trouble:
- Im using Tooltip together with the controllable dialog. When Im opening the Dialog, my Tooltip isn't hide. My Tooltip hides only after some mousemove.

I want to hide my Tooltip after open the Dialog immediately.
Thanks!

**Comment by huuthe0109** [comment]
can u share your repository? I'm facing a problem like this case
